dashboard/config/locales/en.yml

```diff
@@ -431,6 +431,7 @@ en:
       logout: 'Sign out'
       signin: 'Sign in'
       signup: 'Create an account'
+      create_account: 'Create Account'
```

Comment:
similar to when @kelbyhawn added the "Schools" string, I think you also need to add this to the [i18n gsheet](https://docs.google.com/spreadsheets/d/1Tq7VqZALgRA0wYk0HDfEOTyRI0TM2Dir2rloXIPGCgU/edit#gid=0) so that it works on pegasus

Reply:
Roger that!

Reply:
(added)